<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6F1UH3N" failures="1" tests="6" name="Test" time="59.808" errors="0" timestamp="2023-01-22T17:59:26 IST">
  <testcase classname="tests.LoginTest" name="login" time="1.042">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.LoginTest.login(LoginTest.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="tests.LogGeneration2" name="loginGen2" time="1.454"/>
  <testcase classname="tests.LogGeneration2" name="loginGen2" time="11.34"/>
  <testcase classname="tests.LogGeneration" name="loginGen" time="1.368"/>
  <testcase classname="tests.ExampleDataProvider" name="loginData" time="1.513"/>
  <testcase classname="tests.ExampleDataProvider" name="loginData" time="11.28"/>
</testsuite> <!-- Test -->
